xml/docu_styleguide.taglist.xml

```diff
@@ -25,7 +25,7 @@
   respective category.
  </para>
  <para>
-  With Geekodoc 2, it is also possible to use the Internazionalization Tag Set
+  With GeekoDoc 2, it is also possible to use the Internazionalization Tag Set
```

Comment:
Typo:

```suggestion
  With GeekoDoc 2, it is also possible to use the Internationalization Tag Set
```

Reply:
Completely missed this one - thanks so much @tomschr!